# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1495083333 25200
#      Wed May 17 21:55:33 2017 -0700
# Node ID 018fddb4a310116ff864c018aa2fa2d61a46bd14
# Parent  4b38c9687331b260a50576e86e3e397398db092d
maint: drop workarounds for ancient versions of Autoconf

* acinclude.m4 (m4_ifblank, m4_ifnblank): Drop fallback definitions for
ancient versions of Autoconf, Octave now requires Autoconf >= 2.65.
Delete obsolete comment at end of file, file no longer contains external
macro definitions.

diff --git a/m4/acinclude.m4 b/m4/acinclude.m4
--- a/m4/acinclude.m4
+++ b/m4/acinclude.m4
@@ -12,35 +12,16 @@ dnl
 dnl Octave is distributed in the hope that it will be useful, but
 dnl WITHOUT ANY WARRANTY; without even the implied warranty of
 dnl MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 dnl GNU General Public License for more details.
 dnl
 dnl You should have received a copy of the GNU General Public License
 dnl along with Octave; see the file COPYING.  If not, see
 dnl <http://www.gnu.org/licenses/>.
-dnl
-dnl Copyright (C) 2008 - 2009 Free Software Foundation, Inc.
-dnl
-dnl If needed, define the m4_ifblank and m4_ifnblank macros from autoconf 2.64
-dnl This allows us to run with earlier Autoconfs as well.
-dnl FIXME: these should go away once Autoconf 2.64 is required or ubiquitous.
-dnl
-ifdef([m4_ifblank],[],[
-m4_define([m4_ifblank],
-[m4_if(m4_translit([[$1]],  [ ][	][
-]), [], [$2], [$3])])])
-dnl
-ifdef([m4_ifnblank],[],[
-m4_define([m4_ifnblank],
-[m4_if(m4_translit([[$1]],  [ ][	][
-]), [], [$3], [$2])])])
-dnl
-dnl ----------------------------------------------------------------------
-dnl
 
 dnl
 dnl Alphabetical list of macros in the OCTAVE_ namespace
 dnl
 
 dnl
 dnl Check if the Carbon Framework defines CGDisplayBitsPerPixel.
 dnl
@@ -2816,12 +2797,8 @@ AC_DEFUN([OCTAVE_UMFPACK_SEPARATE_SPLIT]
   else
     AC_MSG_RESULT([$octave_cv_umfpack_separate_split])
   fi
   if test $octave_cv_umfpack_separate_split = yes; then
     AC_DEFINE(UMFPACK_SEPARATE_SPLIT, 1,
       [Define to 1 if the UMFPACK Complex solver allows matrix and RHS to be split independently.])
   fi
 ])
-
-dnl         End of macros written by Octave developers
-dnl ------------------------------------------------------------
-dnl
